tests/integration/RequestBuilder/EventsRequestBuilderTest.php

```diff
@@ -6,13 +6,37 @@
 use Lmc\Matej\IntegrationTests\IntegrationTestCase;
 use Lmc\Matej\Model\Command\Interaction;
 use Lmc\Matej\Model\Command\ItemProperty;
+use Lmc\Matej\Model\Command\ItemPropertySetup;
 use Lmc\Matej\Model\Command\UserMerge;
+use Lmc\Matej\RequestBuilder\ItemPropertiesSetupRequestBuilder;
 
 /**
  * @covers \Lmc\Matej\RequestBuilder\EventsRequestBuilder
  */
 class EventsRequestBuilderTest extends IntegrationTestCase
 {
+    private function buildAndSendPropertySetupRequest(ItemPropertiesSetupRequestBuilder $builder): void
+    {
+        $builder->addProperty(ItemPropertySetup::string('test_property_a'))
+            ->addProperty(ItemPropertySetup::string('test_property_b'))
+            ->addProperty(ItemPropertySetup::string('test_property_c'))
+            ->send();
+    }
+
+    protected function setup(): void
```

Comment:
`setup` and `tearDown` methods should be first.

And helper method `buildAndSendPropertySetupRequest` should be at the end of test class.

Reply:
:+1: will fix, thanks